packages/perseus/src/widgets/__tests__/numeric-input.test.ts

```diff
@@ -515,11 +515,8 @@ describe("Numeric input widget", () => {
         const gotFocus = await act(() => renderer.focus());
 
         // Assert
-        expect(gotFocus).toBeTrue();
```

Comment:
`.toBeTrue()` is part of the `jest-extended` package. It sees very low usage in our codebase and `.toBe(true)` from Jest works the same. 

https://khanacademy.atlassian.net/browse/LEMS-2176